pages/howto/easybuild.md

```diff
@@ -17,14 +17,14 @@ For example, the module `java/11.0.16` defines:
 
 You can see the environment variables defined by the `java/11.0.16` module using:  
 ```
-ccruser@vortex ~:$ module show java/11.0.16 |grep EB  
+CCRusername@login ~:$ module show java/11.0.16 |grep EB  
```

Comment:
CCRusername@login:~$ module show java/11.0.16 | grep EB  

Added space after |
Changed prompt to @login:~$, would suggest a commit wide find/replace from " ~:$" to ":~$"

Reply:
fixed formatting.  Space after | isn't necessary